db/migrations/20190104111051-add-name-columns.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('SlackUsers', 'githubName', {
+      type: Sequelize.STRING,
+    });
```

Comment:
Is this `SlackUsers` column used for anything? I can't seem to find it in the rest of the changes.

Reply:
You are right. Deleting this.